src/pages/requests/index.tsx

```diff
@@ -12,7 +12,7 @@ import { Sidebar } from "../../others/components/Sidebar";
 import { MultiTab } from "../../others/components/MultiTab";
 import { CollapsibleTable } from "../../others/components/CollapsibleList";
 import { layerStyle } from "../../others/components/map/CircleLayerStyle";
-import { aidRequestsFixture } from "../../others/fixtures/request.fixture";
```

Comment:
Maybe we could remove this fixture as we are using real data now?

Reply:
Thanks, removed!